ArticleTemplates/assets/scss/modules/_tags.scss

```diff
@@ -46,43 +43,50 @@ Display which keywords an article has been tagged with.
         display: inline-block;
     }
 
-    .inline-list {
-        @include mq($from: col4) {
-            margin-left: 232px; // 240 - 4 (item padding offset) - 4 (visually offset rounded items);
-            width: 940px;
-        }
+    .witness + & {
+        margin-top: base-px(2);
+    }
+}
 
-        &__item {
-            margin: 1rem 0;
-            a {
-                white-space: nowrap;
-                padding: base-px(.3, 1.3, .7, 1.3);
-                border-radius: 30px;
-                margin: 4px;
-                text-overflow: ellipsis;
-                max-width: 18.75rem;
-                @include tag-button(color(shade-1), color(global-tags));
-            }
-            &.more-button a {
-                background-color: transparent !important;
-                @include tag-more-button(color(shade-1), darken(color(global-tags), 10%));
-            }
+.tags__inline-list {
+    display: none;
+}
 
-            &:first-of-type {
-                font-weight: bold;
-            }
+.tags__inline-list:not(:empty) {
+    display: block;
+    border-bottom: 2px solid color(shade-4);
+    border-top: 1px dotted color(shade-4);
+    padding: base-px(.4, 1, 1.7, .7);
 
-            &:last-child {
-                .tags__separator {
-                    display: none;
-                    visibility: hidden;
-                }
-            }
-        }
+    @include mq($from: col4) {
+        margin-left: 232px; // 240 - 4 (item padding offset) - 4 (visually offset rounded items);
+        width: 940px;
     }
+}
 
-    .witness + & {
-        margin-top: base-px(2);
+.tags__list-item {
```

Comment:
Thanks for updating these class names to be syntactically correct BEM names! This will however mean we need to update https://github.com/guardian/mobile-apps-article-templates/blob/master/ArticleTemplates/assets/js/modules/more-tags.js as the old class names are used in the JS. There are some corresponding unit tests for the more-tags module too which will need updating.